modules/digiTrustIdSystem.js

```diff
@@ -9,11 +9,13 @@
  * @requires module:modules/userId
  */
 
-// import { config } from 'src/config';
 import * as utils from '../src/utils'
 import { ajax } from '../src/ajax';
 import { submodule } from '../src/hook';
-// import { getGlobal } from 'src/prebidGlobal';
+import { attachIdSystem } from '../modules/userId/index';
```

Comment:
delete the import for attachIdSystem (attachIdSystem was replaced by submodule)

Reply:
I have made the indicated changes. Please review and LMK if anything else is needed.